src/pages/ProductsByCategory/index.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { useMainContext } from "../../context/main-context";
+import Card from "../../components/Card";
+import { useParams } from "react-router";
+
+export default function ProductsByCategory() {
+	const { state } = useMainContext();
+	const { categoryName } = useParams();
+	console.log("categoryName", categoryName);
```

Comment:
Remove console.log() from the code everywhere

Reply:
done